doc/getting-started/getting-started.rst

```diff
@@ -170,10 +181,10 @@ To set up the ACRN build environment on the development computer:
       cd ~/acrn-work
       git clone https://github.com/projectacrn/acrn-hypervisor.git
       cd acrn-hypervisor
-      git checkout v2.7
+      git checkout master # TODO determine what to check out
 
       cd ..
-      git clone --depth 1 --branch release_2.7 https://github.com/projectacrn/acrn-kernel.git
+      git clone --depth 1 --branch release_2.7 https://github.com/projectacrn/acrn-kernel.git # TODO determine what to check out
```

Comment:
eventually the acrn-kernel branch name would be release_3.0

Reply:
We could use latest stable tag "acrn-2022w18.4-180000p" for now.

Reply:
Done.